src/App.vue

```diff
@@ -0,0 +1,337 @@
+
+        this.insertTile('right')
+        setTimeout( () => {
+          this.moveColumn(2, 'left')
+          this.moveColumn(3, 'left')
+          this.moveColumn(4, 'left')
+          this.moveColumn(5, 'left')
+          setTimeout(() => this.score = findScoreIfGameOver(this.tiles), TRANSITION_INTERVAL * 2 + 5)
+        }, 15)
+
+      } else if (event.code === 'ArrowUp') {
+
+        this.insertTile('bottom')
+        setTimeout( () => {
+          this.moveRow(2, 'up')
+          this.moveRow(3, 'up')
+          this.moveRow(4, 'up')
+          this.moveRow(5, 'up')
+          setTimeout(() => this.score = findScoreIfGameOver(this.tiles), TRANSITION_INTERVAL * 2 + 5)
+        }, 15)
+
+      } else if (event.code === 'ArrowDown') {
+
+        this.insertTile('top')
+        setTimeout( () => {
+          this.moveRow(3, 'down')
+          this.moveRow(2, 'down')
+          this.moveRow(1, 'down')
+          this.moveRow(0, 'down')
+          setTimeout(() => this.score = findScoreIfGameOver(this.tiles), TRANSITION_INTERVAL * 2 + 5)
+        }, 15)
+      } else if (event.code === 'Space') {
+
+        this.newGame()
+      }
+    },
+
+    combineIfPossible (movingTile, targetTile, increment, direction) {
+      const num1 = movingTile.number
+      const num2 = targetTile.number
+
+      if ((num1 === num2 && num1 > 2) || (num1 + num2 === 3)) {
+
+        if (direction === 'horizontal') {
+          movingTile.top = true
+          movingTile.x = movingTile.x + increment
+        } else if (direction === 'vertical') {
+          movingTile.top = true
+          movingTile.y = movingTile.y + increment
+        }
+
+        setTimeout(() => {
+            setTimeout(() => {
+              this.tiles.splice(this.tiles.indexOf(targetTile), 1)
+              movingTile.top = false
+            }, TRANSITION_INTERVAL)
+            movingTile.number = num1 + num2
+          },
+          TRANSITION_INTERVAL
+        )
+      }
+    },
+
+    moveColumn: function (colNum, direction) {
+      const increment = direction === 'right' ? 1 : -1
+
+      const tilesInCol = this.tiles.filter(tile => tile.x === colNum)
+
+      tilesInCol.forEach(tile => {
+        const targetTile = getTileByPosition(tile.x + increment, tile.y, this.tiles)
+        if (!targetTile) {
+          tile.x = tile.x + increment
+        } else {
+          this.combineIfPossible(tile, targetTile, increment, 'horizontal')
+        }
+      })
+    },
+
+    moveRow: function (rowNum, direction) {
+      const increment = direction === 'down' ? 1 : -1
+
+      const tilesInRow = this.tiles.filter(tile => tile.y === rowNum)
+
+      tilesInRow.forEach(tile => {
+        const targetTile = getTileByPosition(tile.x , tile.y + increment, this.tiles)
+        if (!targetTile) {
+          tile.y = tile.y + increment
+        } else {
+          this.combineIfPossible(tile, targetTile, increment, 'vertical')
+        }
+      })
+    }
+  }
+}
+
+</script>
+
+<style>
```

Comment:
same with these styles. ideally all of your CSS would be in a single included bundled asset